src/core/p5.Renderer.js

```diff
@@ -337,369 +337,52 @@ class Renderer {
     }
   }
 
-  textSize(s) {
-    if (typeof s === 'number') {
-      this.states.textSize = s;
-      if (!this.states.leadingSet) {
-      // only use a default value if not previously set (#5181)
-        this.states.textLeading = s * constants._DEFAULT_LEADMULT;
-      }
-      return this._applyTextProperties();
-    }
-
-    return this.states.textSize;
+  _applyDefaults() {
+    return this;
   }
 
-  textLeading (l) {
-    if (typeof l === 'number') {
-      this.states.leadingSet = true;
-      this.states.textLeading = l;
-      return this._pInst;
-    }
-
-    return this.states.textLeading;
+  /**
+   * Helper function to check font type (system or otf)
+   */
+  _hasGlyphData({ font: f } = this.states.textFont) {
+    return typeof f === 'object' && f.data;
   }
 
-  textStyle (s) {
-    if (s) {
-      if (
-        s === constants.NORMAL ||
-        s === constants.ITALIC ||
-        s === constants.BOLD ||
-        s === constants.BOLDITALIC
-      ) {
-        this.states.fontStyle = s;
-      }
+  // old typography functions below
 
-      return this._applyTextProperties();
-    }
-
-    return this.states.fontStyle;
+  textSize() {
+    throw Error('use src/type/textCore.js');
```

Comment:
Are these to indicate to contributors where the actual implementation of these methods live?

Reply:
just placeholders to be removed ... I wanted to make sure those weren't being called in code from somewhere else like `_isOpenType()`...